src/RollbarClientSubmitter.mts

```diff
@@ -94,29 +110,23 @@ function submitOccurrence(url, payload) {
     headers: { 'Content-Type': 'application/json' },
     keepalive: true,
     method: 'POST',
-  });
+  }).catch(() => {});
```

Comment:
I was thinking that since this is an error reporting library, and if connectivity isn't working, there's nothing that can really be done. I suppose adding a console log would be helpful, but it would never reach a reporting service. What do you suggest I do here?

Reply:
I mostly answered with the comment above, I think you are right in most production instances the error is unlikely to be useful, but silencing errors implicitly can also be a dangerous path.
I think we can probably mostly have the discussion up there, I added my arguments but I don't have a super strong opinion as I think the usefulness of those errors would be a rare case.

Reply:
Handled; see above

Reply:
The `catch` still seems to be here, and if I have understood correctly the chain of actions within the library, this is the deepest in the call stack, hence catching here will swallow all errors, or am I missing something?

Reply:
No you're right. I missed the most important one! 🙃

Thank you for noticing. All fixed now. I had to refactor a bunch of tests because I had to `await fetch` and thus make the containing function `async`, and that's been done too.